test/w_supertensor.cpp

```diff
@@ -592,4 +592,8 @@ struct superTensorTest : test_driver
     }
 };
 
-int main(int argc, const char* argv[]) { test_drive<superTensorTest>(argc, argv); }
+int main(int argc, const char* argv[])
+{
+    test_drive<superTensorTest>(argc, argv);
+    return 0;
```

Comment:
I dont think this is necessary, `main()` should return `0` without needing an explicit return statement, From [cppreference](https://en.cppreference.com/w/cpp/language/main_function):

> The body of the main function does not need to contain the return statement: if control reaches the end of main without encountering a return statement, the effect is that of executing return 0;.

Reply:
Never thought about this ;)